src/plugins/intel_gpu/include/intel_gpu/primitives/convolution.hpp

```diff
@@ -763,6 +763,20 @@ struct convolution : public primitive_base<convolution> {
     /// @brief List of primitive ids containing compensation.
     primitive_id_arr compensation;
 
+    size_t hash() const override {
+        if (!seed) {
+            seed = hash_range(seed, pad.begin(), pad.end());
```

Comment:
I guess number of inputs and outputs must be a part of hash as it may impact mode of work for some kernels (conv w/ and w/o bias, concat, etc)

Reply:
Add number of inputs / outputs to hash functions